packages/url_launcher/url_launcher_platform_interface/test/link_test.dart

```diff
@@ -83,6 +83,9 @@ class _RouteDelegate extends RouterDelegate<RouteInformation>
     if (_history.isEmpty) {
       return const Placeholder(key: Key('empty'));
     }
+    // TODO(chunhtai): remove the ignore once migration completes.
```

Comment:
Could you summarize in the comment what "once migration completes" means concretely? The issue you linked to is really long, so it's not immediately obvious.